README.md

```diff
@@ -43,20 +72,15 @@ All the most important links for TAG Environmental Sustainability are also avail
   * Twitter/X: [@CNCFEnvTAG](https://twitter.com/CNCFEnvTAG)
   * YouTube: [@CNCFEnvTAG](https://www.youtube.com/@CNCFEnvTAG)
 
-### TAG Environmental Sustainability Co-Chairs
+### Add TAG ENV meeting Calendar to your calendar
 
-* [Marlow Weston](https://github.com/catblade)
-* [Leonard Vincent Simon Pahlke](https://github.com/leonardpahlke)
+Import the TAG ENV Calendar feed into your calendar from following URL: [TAG ENV Calendar](https://calendar.google.com/calendar/embed?src=72e93a411f02e5664bb4485c04311b83dae6a62574e4ab882a1ccf8526aa9bf1%40group.calendar.google.com).
+[Subscribe to someone’s Google Calendar](https://support.google.com/calendar/answer/37100?hl=en&co=GENIE.Platform%3DDesktop). In this case your calendar will be automatically updated once new events are published.
 
-### TAG Environmental Sustainability Tech Leads
+## Roadmap
 
-* [Kristina Devochko](https://github.com/guidemetothemoon)
-* [Saiyam Pathak ](https://github.com/saiyam1814)
+Currently we don't have a dedicated roadmap document, but we have a [GitHub issue](https://github.com/cncf/tag-env-sustainability/issues/560) to track the work related to its creation. Once the document is created, this section will be updated.
```

Comment:
We could ref the GitHub project board here. But, yes we do not have a roadmap document right now. There is a roadmap for wg Green reviews & wg advocacy is working on one too.

Reply:
Thanks, Leo! Added a sentence with the link to the project board.